webrtc.html

```diff
@@ -8713,6 +8713,15 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           values in <var>configuration</var>.</p>
         </li>
         <li>
+          <p>Initialize <var>channel</var>'s
+          <a data-for="RTCDataChannel"><code>priority</code></a> attribute to
+          the <a>RTCDataChannelPriority</a> enum value that corresponds to the
+          integer priority value in <var>configuration</var>, as defined in
```

Comment:
I'm not sure of that. _configuration_ is not exactly a JavaScript object and there is no type definition of what fields and types it has. Step 3 only says:

> Let configuration be an information bundle received from the other peer as a part of the process to establish the underlying data transport described by the WebRTC DataChannel Protocol specification [RTCWEB-DATA-PROTOCOL].

If _configuration.priority_ already have the `RTCDataChannelPriority` type, we'd be able to just set the value directly, and there wouldn't need to have so much explanation on how to construct `channel.priority` from _configuration_.